tests/unit/dumm_env

```diff
@@ -6,5 +6,5 @@ GITHUB_CLIENT_SECRET=rezindkm3dod04h12yt8oktm8if51wtsfgljk04k
 JIRA_INSTANCE="https://my-jira.atlassian.net"
 JIRA_USERNAME=maksim.beliaev@canonical.com
 JIRA_TOKEN=mv38swy07r6ius3v90cffyi4
-DEFAULT_BOT_CONFIG='{"settings": {"components": null, "labels": ["jira"], "add_gh_comment": false, "sync_description": true, "sync_comments": true, "epic_key": null, "jira_project_key": null, "label_mapping": null, "status_mapping": null}}'
+DEFAULT_BOT_CONFIG='{"settings": {"components": null, "labels": ["jira"], "add_gh_comment": false, "add_gh_synced_label": false, "sync_description": true, "sync_comments": true, "epic_key": null, "jira_project_key": null, "label_mapping": null, "status_mapping": null}}'
```

Comment:
let's remove this change, that would be a better test for existing environments where this flag is not added

Reply:
Unless we add a try-except block around the `if settings["add_gh_synced_label"]:` statement then the old tests will actually fail without this option set to false.

Reply:
that is good that we caught it right here, since this is backwards incompatibility.

why not to use  `if settings.get("add_gh_synced_label", False)`  then?